packages/vaadin-grid/test/drag-and-drop.test.js

```diff
@@ -753,7 +753,7 @@ describe('drag and drop', () => {
   describe('auto scroll', () => {
     beforeEach(async () => {
       grid.dropMode = 'between';
-      grid.items = Array(...new Array(100)).map(() => {});
+      grid.items = new Array(100).fill({});
```

Comment:
It doesn't lead to the same result, I would guess. After the change, all the grid items are going to be represented by a **single** object, which was not so before.

Reply:
The previous array contained `undefined` because of improver usage of arrow function in `.map` 🤷‍♂️